docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,884 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ae11dc9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# some basic imports\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from power_grid_model import LoadGenType\n",
+    "from power_grid_model import PowerGridModel, CalculationMethod, CalculationType, MeasuredTerminalType\n",
+    "from power_grid_model import initialize_array"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f983cef7",
+   "metadata": {},
+   "source": [
+    "## Power Flow Calculation\n",
+    "### Input Dataset for State Estimation\n",
+    "\n",
+    "We create input dataset by using the helper function `initialize_array`.\n",
+    "Note the units of all input are standard SI unit without any prefix,\n",
+    "i.e. the unit of voltage is volt (V), not kV.\n",
+    "\n",
+    "Please refer [Components](https://power-grid-model.readthedocs.io/en/stable/user_manual/components.html) for detailed explanation of all component types and their input/output attributes. \n",
+    "\n",
+    "**NOTE: The required attributes of each components can be different for asymmetric calculations.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6f008736",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# node\n",
+    "node = initialize_array(\"input\", \"node\", 3)\n",
+    "node[\"id\"] = np.array([1, 2, 6])\n",
+    "node[\"u_rated\"] = [10.5e3, 10.5e3, 10.5e3]\n",
+    "\n",
+    "# line\n",
+    "line = initialize_array(\"input\", \"line\", 3)\n",
+    "line[\"id\"] = [3, 5, 8]\n",
+    "line[\"from_node\"] = [1, 2, 1]\n",
+    "line[\"to_node\"] = [2, 6, 6]\n",
+    "line[\"from_status\"] = [1, 1, 1]\n",
+    "line[\"to_status\"] = [1, 1, 1]\n",
+    "line[\"r1\"] = [0.25, 0.25, 0.25]\n",
+    "line[\"x1\"] = [0.2, 0.2, 0.2]\n",
+    "line[\"c1\"] = [10e-6, 10e-6, 10e-6]\n",
+    "line[\"tan1\"] = [0.0, 0.0, 0.0]\n",
+    "line[\"i_n\"] = [1000, 1000, 1000]\n",
+    "line[\"r0\"] = [0.25, 0.25, 0.25]\n",
+    "line[\"x0\"] = [0.2, 0.2, 0.2]\n",
+    "line[\"c0\"] = [10e-6, 10e-6, 10e-6]\n",
+    "line[\"tan0\"] = [0, 0, 0]\n",
+    "\n",
+    "# sym load\n",
+    "sym_load = initialize_array(\"input\", \"sym_load\", 1)\n",
+    "sym_load[\"id\"] = [4]\n",
+    "sym_load[\"node\"] = [2]\n",
+    "sym_load[\"status\"] = [1]\n",
+    "sym_load[\"type\"] = [LoadGenType.const_power]\n",
+    "sym_load[\"p_specified\"] = [20e6]\n",
+    "sym_load[\"q_specified\"] = [5e6]\n",
+    "\n",
+    "# asym load\n",
+    "asym_load = initialize_array(\"input\", \"asym_load\", 1)\n",
+    "asym_load[\"id\"] = [7]\n",
+    "asym_load[\"node\"] = [6]\n",
+    "asym_load[\"status\"] = [1]\n",
+    "asym_load[\"type\"] = [LoadGenType.const_power]\n",
+    "asym_load[\"p_specified\"] = [[10e6, 20e6 , 0]]\n",
+    "asym_load[\"q_specified\"] = [[0, 8e6, 2e6]]\n",
```

Comment:
maybe you can add an inline comment here that highlights the difference compared to `sym_loads`, e.g.

```suggestion
    "asym_load[\"p_specified\"] = [[10e6, 20e6 , 0]] # the 3 phases may have different loads\n",
    "asym_load[\"q_specified\"] = [[0, 8e6, 2e6]]    # the 3 phases may have different loads\n",
```